app/models/agents/telegram_agent.rb

```diff
@@ -0,0 +1,96 @@
+require 'httmultiparty'
+require 'open-uri'
+require 'tempfile'
+
+module Agents
+  class TelegramAgent < Agent
+    cannot_be_scheduled!
+    cannot_create_events!
+    no_bulk_receive!
+
+    description <<-MD
+      The Telegram Agent receives and collects events and sends them via [Telegram](https://telegram.org/).
+
+      It is assumed that events have either a `text`, `photo`, `audio`, `document` or `video` key. You can use the EventFormattingAgent if your event does not provide these keys.
+
+      The value of `text` key is sent as a plain text message.
+      The value of `photo`, `audio`, `document` and `video` keys should be a url whose contents will be sent to you.
+
+      **Setup**
+
+      1. Obtain an `auth_token` by [creating a new bot](https://telegram.me/botfather).
+      2. Send a private message to your bot by visiting https://telegram.me/YourHuginnBot
+      3. Obtain your private `chat_id` from the recently started conversation by visiting https://api.telegram.org/bot<auth_token>/getUpdates
```

Comment:
Okay, that is why I was confused. The description contains the auth_token, but markdown seems to omit it! Thank you for reporting that!